ui/src/components/Selection.tsx

```diff
@@ -147,6 +147,29 @@ const EditLabelModal = ({ annotation, onHide }: EditLabelModalProps) => {
         e.stopPropagation();
     };
 
+    useEffect(() => {
+        const onKeyPress = (e: KeyboardEvent) => {
+            // Ref to https://github.com/allenai/pawls/blob/0f3e5153241502eb68e46f582ed4b28112e2f765/ui/src/components/sidebar/Labels.tsx#L20
+            // Numeric keys 1-9
+            if (e.keyCode >= 49 && e.keyCode <= 57) {
+                const index = Number.parseInt(e.key) - 1;
+                if (index < annotationStore.labels.length) {
+                    const selectedLabel = annotationStore.labels[index];
+                    annotationStore.setPdfAnnotations(
+                        annotationStore.pdfAnnotations
+                            .deleteAnnotation(annotation)
+                            .withNewAnnotation(annotation.update({ label: selectedLabel }))
+                    );
+                    onHide();
+                }
+            }
+        };
+        window.addEventListener('keydown', onKeyPress);
```

Comment:
Would this conflict with https://github.com/allenai/pawls/blob/0f3e5153241502eb68e46f582ed4b28112e2f765/ui/src/components/sidebar/Labels.tsx#L46 ?

Reply:
It depends on what you mean by conflict.

In this situation they're both listening to the keydown event, and listening for the same set of keys. So both will fire when the respective keys are pressed.  

So I think after this change, when a user presses a number key:

1. The selected label (in the sidebar) will change.
2. *Every* annotation's label will be updated to the one that's now selected.

If that's desired...maybe that's ok?

Reply:
Thanks! Yes, that's what I meant by "conflict" -- they'll be called simultaneously. 

> The selected label (in the sidebar) will change

▲ I think it somehow makes sense, right? Because when we create a new annotation with a different category, we need to change the category on the sidebar first. When modifying the annotation categories, it's reasonable that the current selected category is modified according to the (new) selected label. 


> Every annotation's label will be updated to the one that's now selected.

▲ I think right now only one annotation can be selected and modified? If that's the case, I think it is probably the desired function.

Reply:
> I think it somehow makes sense, right? Because when we create a new annotation with a different category, we need to change the category on the sidebar first. When modifying the annotation categories, it's reasonable that the current selected category is modified to the (new) selected label.

Maybe. My gut says only one the active annotation's label should change -- but it's an easy thing to tweak later, and I'd have to play around with it a bit more to be sure. So let's give this treatment a shot.

> I think right now only one annotation can be selected and modified? If that's the case, I think it is probably the desired function.

Correct. I hadn't read through the code, but you helped me see that this is clearly the case. Thanks!